src/chapters/aspdotnet-model-validation/viewmodels.rst

```diff
@@ -0,0 +1,91 @@
+.. index:: ! ViewModel
+
+ViewModels and Passing Data To and From Views
```

Comment:
I like the overview/motivation for using ViewModels here. One additional important motivation is that a form in a view may require different data than the model uses. A common example of this is a user registration form, which requires a password field and a verify password field. The user model, however, would only have a password field (or a passwordHash field).

Reply:
Added that example. Thanks!